lib/js/app/components/ActionsMenu/ActionsMenu.tsx

```diff
@@ -51,15 +56,15 @@ const ActionsMenu: FC<Props> = ({ isNewQuery, onRemoveQuery, onHideMenu }) => {
           onMouseEnter={() => !queryResults && showTooltip(true)}
           onMouseLeave={() => tooltip && showTooltip(false)}
         >
-          <AnimatePresence>
-            {tooltip && (
+          {tooltip && (
```

Comment:
Based on `framer-motion` docs the _<AnimatePresence />_ component should be mounted and conditionally render it children. By this change we'll always create a new instance of Framer <AnimatePresence /> component if tooltip is visible.